### PR TITLE
feat: Hunger Games on Facets

### LINKED ISSUE #2804: Integrate Hunger Games on the category, labels and brands pages
### What
- Now that we can category target Hunger Games to categories, a link on the category, label and brand page for logged in users would be nice
- Note: the website layout is about to change
### Screenshot of the Power User Script implementation
image

### Code pointers
- https://hunger.openfoodfacts.org/questions?type=category&value_tag=en%3Aalcoholic-beverages
- https://github.com/openfoodfacts/power-user-script
```
<sup><a class="button tiny round secondary label" href="https://hunger.openfoodfacts.org/questions?type=category&amp;value_tag=en:crystallized-ginger">Help categorize more of these</a></sup>
```

### PR DESCRIPTION
**Pull request by teolemon**
### What

- Hunger Games on Facets for Categories, Labels and Brands.
- Will create a deep link to Hunger Games for that specific value of the facet

### Related issue(s) and discussion

- Fixes #2804

### CODE REVIEW

**Comment by teolemon** [comment]
@stephanegigandet Could you point me from which perl file I should inject tag_type into the template ?

**Comment by stephanegigandet** [comment]
> @stephanegigandet Could you point me from which perl file I should inject tag_type into the template ?

I added the tagtype and canon_tagid.

File: lib/ProductOpener/Display.pm [hunk 1]
```diff
@@ -7032,6 +7036,8 @@ JS
 
 	$template_data_ref->{scripts} = $scripts;
 	$template_data_ref->{initjs} = $initjs;
+
```
Comment (teolemon) [inline lib/ProductOpener/Display.pm:7039]:
```suggestion
```

**Comment by teolemon** [comment]
tests now pass @stephanegigandet

**Review by stephanegigandet — APPROVED** [review]

**Pull request merged**